### PR TITLE
Add Styling on the react project

### PR DESCRIPTION
**Pull request by BonkeGcobo**
Adding Styling on the project

### CODE REVIEW

File: src/components/Books/ProgressBar.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  return (
+    <div className="user-progress-container">
+      <div className="user-progress">
+        <div
+          className="progressbar"
+          style={{
+            width: '4.25rem',
+            height: '4.25rem',
+          }}
+        >
```
Comment (AshakaE) [inline src/components/Books/ProgressBar.js:25]: - [x] Please note that inline styling is not allowed for this project

**Review by AshakaE — CHANGES_REQUESTED** [review]
Hi @BonkeGcobo , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

- [x] Please note that styling is exactly the same as the one in the [Zeplin template](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378), that includes (fonts face, font size and spacing)

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Books/ProgressBar.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+  return (
+    <div className="user-progress-container">
+      <div className="user-progress">
+        <div
+          className="progressbar"
+          style={{
+            width: '4.25rem',
+            height: '4.25rem',
+          }}
```
Comment (Mihndim2020) [inline src/components/Books/ProgressBar.js:24]: - [OPTIONAL] As requested by the previous code reviewer, please kindly define these styles in a different file and import them in to this component, for inline styling is not a best practice.

File: src/components/Books/ProgressBar.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+        >
+          <CircularProgressbar
+            value={percentage}
+            styles={buildStyles({
+              pathColor: '#379cf6',
+              trailColor: '#e8e8e8',
+            })}
```
Comment (Mihndim2020) [inline src/components/Books/ProgressBar.js:31]: - [OPTIONAL] As requested by the previous code reviewer, please kindly define these styles in a different file and import them in to this component, for inline styling is not a best practice.

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @BonkeG, 

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**